Plugins/Animation/src/AnimationComponent.cpp

```diff
@@ -393,4 +378,55 @@ Scalar AnimationComponent::getTime() const {
     return m_animationTime;
 }
 
+Scalar AnimationComponent::getDuration() const {
+    if ( m_animations.empty() )
+    {
+        return 0;
+    }
+    return m_animations[m_animationID].getDuration();
+}
+
+void AnimationComponent::cacheFrame( int frame ) const {
+    std::ofstream file( "animFrames/"+m_contentName+"_frame"+std::to_string(frame)+".anim",
```

Comment:
Doesn't Radium has a standard repository for runtime file outputs ?

Reply:
Actually, these runtime outputs are not relevant only for the current session, it's more like added data to the scene.
I will change that and have the user specify where he wants to save these data (better be along the scene file)